common/djangoapps/third_party_auth/migrations/0002_auto_20200721_1650.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.14 on 2020-07-21 16:50
```

Comment:
whoever merges last between this and [my pr](https://github.com/edx/edx-platform/pull/24560) will probably have to rebase just as a heads up.

Reply:
yes I commented the same on your pr yesterday.

Reply:
ah I did not actually comment that on your PR, just commented on it in the slack channel. My bad. I don't doubt yours will go first since I have test failures and yours appear to be passing.